compiler/luci-interpreter/src/kernels/Tanh.test.cpp

```diff
@@ -97,6 +97,52 @@ TEST(TanhTest, Uint8)
   EXPECT_THAT(extractTensorShape(output_tensor), ::testing::ElementsAreArray(ref_output_shape));
 }
 
+TEST(TanhTest, InputTypeInvalid_NEG)
```

Comment:
A little Question. This NEG case is handled on `LUCI_INTERPRETER_CHECK(input()->element_type() == output()->element_type());` which means input and output mismatching. and on 'tensorflow/lite/kernels/activation.cc', there's no type check about input.

Reply:
You are right, @struss ,
I was meant to do `EXPECT_ANY_THROW(kernel.execute());`, but I missed this part.
I will edit and recommit this.
Thank you for your advice, @struss  :)